mysql-test/suite/sys_vars/t/binlog_row_event_max_size.test

```diff
@@ -0,0 +1,11 @@
+--echo #
+--echo # MDEV-30188 Show "--binlog_row_event_max_size" state in SYSTEM VARIABLES
+--echo #
+
+SELECT @@binlog_row_event_max_size AS EXPECT_8192;
+
```

Comment:
Can you also add a quick test showing it is read only? (Once you update the variable).